mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -816,15 +815,18 @@ class PopulateParamsXDL : public PopulateParamsBase {
   // clang-format off
   llvm::SmallVector<InitParamsXDL, 4> initParametersForwardI8 = {
       // M/block N/block K/block M/wave N/wave kPack aCopyMore bCopyMore
-      // TODO remove
-      {64, 64, 1, 32, 32, 1, false, false},
-
-      {64, 64, 1, 32, 32, 16, false, false},
-      {64, 64, 2, 32, 32, 16, false, false},
-      {64, 64, 4, 32, 32, 16, false, false},
-
-      // TODO Amend all kPack to be 16 for i8
-      {16, 16, 16, 16, 16, 1, false, false},
+      // TODO enable kpack
```

Comment:
Do we need to change the heuristic values for non-XDLOPS path? Or are we concerned about XDLOPS path for now?

Reply:
Good point, actually non-xdlops doesn't need to change because those tuning parameters are tailored for the two xdlops instructions for i8 only (and non-xdlops doesn't have this issue).

On the other hand non-xdlops nightly is already all-passing with the existing tuning parameters.